### DISCUSSION TITLE
Why is student verification so difficult

### DISCUSSION THREAD

**Discussion by jhayden02** [community/community#116466] [GitHub Education]
### Hi everyone,

I am a student in college and attempting to verify my status. My ID doesn't have a date on it so I'm attempting to use one of the many other options listed. My college allows an enrollment verification document to be released to you whenever you request, signed by the registrar. It would be helpful if I had some way to actually send this eDoc. Instead, the only option given is to use my webcam to send this document. The page itself even says you can use your camera or send a document. How exactly would I do that if camera is the only option. You have me logging in on my phone trying to snap a blurry front camera picture of my desktop (the back camera option on phone doesn't work either for Christ's sake) which most likely will be not accepted due to a "blurry photo". Why is this so difficult for no reason. Just allow me to send eDocs- no other place I've verified my studentship on have ever not allowed submitted an eDocument.

![image](https://github.com/community/community/assets/129789424/6b3a076f-2bf2-4e7b-9cc7-0abf4ee0e6a4)

**Comment by mishmanners** [comment]
The same is resolved here: https://github.com/orgs/community/discussions/88401#discussioncomment-8142514 and here: https://github.com/orgs/community/discussions/88446#discussioncomment-8148738

Please see these. The TL;DR:

- Refresh the page or clear your browser's cache and cookies
- Check for browser extensions that could be interfering with the JavaScript code and disable them
- Try a different browser
- Ensure your file is less than 150kB

**Comment by moozunch** [comment]
Hi, same here hehe, my ID doesn't have date on it and my camera be blurry asf but i still get accepted anyway, i think Github have their own listing of school of which allowed without date in their school ID so try to take a request anyway, hope this help ^__________^
